vm/systemSmartContracts/auction.go

```diff
@@ -492,8 +542,16 @@ func (s *stakingAuctionSC) stake(args *vmcommon.ContractCallInput) vmcommon.Retu
 		for i := maxNodesToRun*2 + 1; i < uint64(lenArgs); i++ {
 			if len(args.Arguments[i]) == len(args.CallerAddr) {
 				registrationData.RewardAddress = args.Arguments[i]
+				err = s.eei.UseGas(s.gasCost.BaseOperationCost.StorePerByte * uint64(len(args.Arguments[i])))
```

Comment:
Did I understand this if correctly: 
if we have the length of an address, we have an address in arguments[i] and we set the rewardAddress 
else we have a value as a []byte and we set the MaxStakePerNode.

Could we have a len of MaxStakePerNode equal to an Address? Padding the bytes with 0 or something?

Reply:
Yes. it is correct. But this would be highly unlikely.

Reply:
But there's a chance.